evap/staff/importers.py

```diff
@@ -723,16 +786,32 @@ def process_contributors(self, evaluation, test_run, user_list):
         # makes anything relying on hashes unusable here (for a faster list difference)
         users_to_add = [user for user in user_list if user not in already_related]
 
-        if not test_run:
-            for user in users_to_add:
-                order = Contribution.objects.filter(evaluation=evaluation).count()
-                Contribution.objects.create(evaluation=evaluation, contributor=user, order=order)
-            msg = format_html(_("{} contributors added to the evaluation {}:"), len(users_to_add), evaluation.full_name)
+        if len(users_to_add) == 0:
+            if not test_run:
+                msg = format_html(_("No contributors added to the evaluation {}."), evaluation.full_name)
+            else:
+                msg = format_html(_("No contributors would be added to the evaluation {}."), evaluation.full_name)
         else:
-            msg = format_html(
-                _("{} contributors would be added to the evaluation {}:"), len(users_to_add), evaluation.full_name
-            )
-        msg += create_user_list_html_string_for_message(users_to_add)
+            if not test_run:
+                for user in users_to_add:
+                    order = Contribution.objects.filter(evaluation=evaluation).count()
+                    Contribution.objects.create(evaluation=evaluation, contributor=user, order=order)
```

Comment:
I think this should be outside of the `len(users_to_add) == 0` block, the loop doesn't run if the length is zero anyways and then one would be able to spot / skip the pluralization code better